### DISCUSSION TITLE
Willing to help for a possible French Translation

### DISCUSSION THREAD

**Discussion by Asviix** [sandboxie-plus/Sandboxie#1281] [Q&A Feedback] [answered]
Hello everyone!
First thanks for making such an awesome program, and I've noticed it's not even translated in French!
I'd gladly help if you need me helping translate, just hmu!

Bye !

**Comment by isaak654** [comment, marked as answer]
There is already a French translation, but it's still a working in progress.
I would suggest to reach the following French translator in order to continue the translation: https://github.com/sandboxie-plus/Sandboxie/issues/1155
